lms/djangoapps/courseware/tests/test_views.py

```diff
@@ -191,6 +191,48 @@ def test_jumpto_id_invalid_location(self):
 
 @attr(shard=2)
 @ddt.ddt
+class IndexQueryTestCase(ModuleStoreTestCase):
+    """
+    Tests for query count.
+    """
+    CREATE_USER = False
+    NUM_PROBLEMS = 20
+
+    @ddt.data(
+        (ModuleStoreEnum.Type.mongo, 8),
+        (ModuleStoreEnum.Type.split, 4),
```

Comment:
**Note to reviewers:** On master, instead of `4`, this results in `23` mongo calls for Split and grows linearly with the number of blocks in the course.